db_model/orm_model.py

```diff
@@ -116,6 +116,7 @@ class BuildAction(Base):
     run_id = Column(Integer,
                     ForeignKey('runs.id', deferrable=True, initially="DEFERRED",
                                ondelete='CASCADE'))
+    # hash of the build command
```

Comment:
I think if you pass a string to the first argument of Column, than you can set the name of the column in the database. So you can change the name of the field without changing the database layout.